packages/components/CHANGELOG.md

```diff
@@ -8,6 +8,53 @@ menu: Changelog
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [3.0.0-alpha.0](https://github.com/GetJobber/atlantis/compare/@jobber/components@2.90.0...@jobber/components@3.0.0-alpha.0) (2023-01-24)
+
+
+### Features
+
+* **components:** Update drawer, modal and card headings ([#991](https://github.com/GetJobber/atlantis/issues/991)) ([1d9ade2](https://github.com/GetJobber/atlantis/commit/1d9ade2ec2043c0dcd7b9415409e094fcd7d27cb))
+* **design:** BREAKING CHANGE: Implement new typefaces in Atlantis [JOB-46209] ([#952](https://github.com/GetJobber/atlantis/issues/952)) ([15f604c](https://github.com/GetJobber/atlantis/commit/15f604cc51ee6881a5ed9c1be01081c76cf5ce6e))
+
+
+### BREAKING CHANGES
```

Comment:
Is this a manual changelog edit? 👀 If so, TIL we could do that

Reply:
There is a command to generate it

Reply:
✨